fleet-server.reference.yml

```diff
@@ -141,6 +141,11 @@ fleet:
 #       max_agents: 0
 #       # policy_throttle is the duration that the fleet-server will wait in between attempts to dispatch policy updates to polling agents # TODO verify this
 #       policy_throttle: 5ms # 1ms min is forced
+#       # action_throttle is added as a delay before the in checkin requests to help space response writes
+#       # This is done in order to be able to reuse gzip writers if gzip is requested as allocating new writers is expensive
+#       # should never me more then 2m to prevent edge cases with assumptions around poll-times and server writes from becoming invalid
+#       # a 0 value disables the throttle (default behavior)
+#       action_throttle: 0
```

Comment:
```suggestion
#       # action_throttle is added as a delay before writing checkin responses to help space out response writes
#       # This is done in order to be able to reuse gzip writers if gzip is requested as allocating new writers is expensive
#       # should never be more then 2m to prevent edge cases with assumptions around poll-times and server writes from becoming invalid
#       # a 0 value disables the throttle (default behavior)
#       action_throttle: 0
```

A few typos.